badge_server/main.py

```diff
@@ -104,6 +104,26 @@ class BadgeStatus(enum.Enum):
 }
 
 
+def _get_supported_versions(package_name: str) -> Iterable[int]:
```

Comment:
```suggestion
def _get_supported_versions(package_name: str) -> List[int]:
```

`Iterable` there is not correct because you take the len() on line 146

Reply:
done